src/topp/ProteinQuantifier.cpp

```diff
@@ -595,7 +597,12 @@ class TOPPProteinQuantifier :
     if (params.empty()) params = "(none)";
     else params.resize(params.size() - 2); // remove trailing ", "
     out << "# Parameters (relevant only): " + params << endl;
-
+    string rawfiles;
+    for(vector<String>::const_iterator it=rawfiles_.begin();it!=rawfiles_.end();++it)
```

Comment:
this can be done easier: `ListUtils::concatenate(rawfiles_, "\t");`
see https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Documentation/nightly/html/classOpenMS_1_1ListUtils.html
(also avoids the extra space/tab at the end)